packages/wmr/rollup.config.js

```diff
@@ -109,6 +109,35 @@ const config = {
 				}
 			}
 		},
+		{
+			// This ensures the template files for rollup-plugin-visualizer are inlined
+			// rather than bundleds as fs.readFile()
+			name: 'fix-visualizer',
+			transform(code, id) {
+				if (/rollup-plugin-visualizer[/\\]plugin[/\\]build-stats\.js$/.test(id)) {
+					code = code.replace(
+						/fs\.readFile\(path\.join\(__dirname,\s*(.+?)\)\s*,\s*"utf8"\s*\)/g,
+						(str, stringifiedJoin) => {
+							const path = require('path');
+							const fs = require('fs');
+							const filePathParts = stringifiedJoin
+								.replace(/['"`]+/g, '')
+								.replace(/\$\{template\}/g, 'treemap')
+								.split(', ');
+							const filepath = path.resolve(path.dirname(id), ...filePathParts);
```

Comment:
For simplicity's sake when reviewing, these are the lines that need to be replaced: https://github.com/btd/rollup-plugin-visualizer/blob/799b5337db4fbbf8b83119b6251efe9e33c0ab69/plugin/build-stats.js#L58-L59

The `template` is a visualizer format, i.e., treemap, sunburst, or network. Treemap seems to be the most common so I just went with that.